src/deepsparse/transformers/utils/helpers.py

```diff
@@ -33,11 +33,45 @@
     "override_config",
     "process_generation_config",
     "validate_session_ids",
+    "set_generated_length",
 ]
 
 _LOGGER = logging.getLogger(__name__)
 
 
+def set_generated_length(
+    max_length: int,
+    prompt_tokens_length: int,
+    sequence_length: int,
+    prompt_sequence_length: int,
+    max_new_tokens: Union[int, None],
```

Comment:
Should we default this to None?

Reply:
It'll always be given as an input, but that input might be None (which is why I used the Union but not sure if best practice).